props/saber_fett263_buttons.h

```diff
@@ -1032,6 +1022,325 @@ class GestureControlFile : public ConfigFile {
   int maxclash; // maximum Clash Strength for Clash Sound and Detection works with CLASH_THRESHOLD_G to create range of Clash Strength (8 ~ 16 range)
 };
 
+GestureControlFile saved_gesture_control;
+
+#ifdef MENU_SPEC_TEMPLATE
+template<class SPEC>
+struct SwingGesture : public mode::BoolSetting {
+  bool get() { return saved_gesture_control.swingon;  }
+  void set(bool value) { 
+  saved_gesture_control.swingon = value;
```

Comment:
this and next line should be indented two more steps

Reply:
updated